src/Webfactory/Slimdump/Config/FakerReplacer.php

```diff
@@ -51,8 +52,21 @@ public static function isFakerColumn($replacement)
     public function generateReplacement($replacementId)
     {
         $replacementMethodName = str_replace(self::PREFIX, '', $replacementId);
-        $this->validateReplacementConfigured($replacementMethodName);
-        return $this->faker->$replacementMethodName;
+
+        if (strpos($replacementMethodName,'->') !== false) {
+
+            $modifierAndMethod = explode('->',$replacementMethodName);
+            $modifierName = $modifierAndMethod[0];
+            $replacementMethodName = $modifierAndMethod[1];
+
+            $this->validateReplacementConfiguredModifier($modifierName,$replacementMethodName);
+
+            return (string)$this->faker->$modifierName->$replacementMethodName;
+        }
+        else{
```

Comment:
Please use PSR-2:
- ```} else {``` in one line
- add a single space between a comma and the next parameter